csharp/Clean Code/ToDo/Program.cs

```diff
@@ -5,33 +5,48 @@ namespace ToDo
 {
     internal class Program
     {
-        public static List<string> TL { get; set; }
+        public static List<string> TaskList { get; set; } = new List<string>();
 
         static void Main(string[] args)
         {
-            TL = new List<string>();
-            int variable = 0;
+            const int addTask = 1;
+            const int removeTask = 2; 
+            const int consultTasks = 3; 
+            const int exit = 4;
+            int menuOption = 0;
+
             do
             {
-                variable = ShowMainMenu();
-                if (variable == 1)
-                {
-                    ShowMenuAdd();
-                }
-                else if (variable == 2)
-                {
-                    ShowMenuDos();
-                }
-                else if (variable == 3)
+                menuOption = ShowMainMenu();
+                switch (menuOption)
                 {
-                    ShowMenuTres();
+                    case addTask:
+                        ShowMenuAdd();
+                        break;
+
+                    case removeTask:
+                        ShowMenuRemove();
+                        break;
+
+                    case consultTasks:
+                        ShowMenuConsult();
+                        break;
+
+                    case exit:
+                        Console.WriteLine("See you!");
+                        break;
+
+                    default:
+                        Console.WriteLine("Option not allowed, try again");
+                        break;
                 }
-            } while (variable != 4);
+            } while (menuOption != exit);
         }
+        
         /// <summary>
-        /// Show the main menu 
+        /// Shows the tasks options to redirect to the selected menu
```

Comment:
Regarding comments, I think the general rule is to keep method and variable names meaningful and descriptive and avoid doing code that needs to be explained in a comment, so for the most part of your code you wanna avoid commenting things and instead focus on making it very simple (in all possible ways). For the times where code is not simple/clear enough, is a great idea to add comments to help the reader